docker/nginx/locations

```diff
@@ -3,6 +3,8 @@
 
     location /assets/about {
         # Requests to this location should not fall back to index.html
+        error_page 404 /non_existent_page.html;
+        error_page 403 /non_existent_page.html;
```

Comment:
```suggestion
        # Requesting existing files in this location should not fall back to index.html.
        # Files that do not exist should be handled by index.html
        error_page 404 /non_existent_page.html;
```

Right now I don't see a reason to add the `403`, proof me wrong.

Reply:
I just found out, `error_page 404 /index.html;` also works, so would not need `non_existent_page.html`.

Also, found out when 403 seems to come, when accessing e.g. `/assets`.
I would propose to use:


```
error_page 403 /index.html;
error_page 404 /index.html;

```
Before all location blocks.

Reply:
Yes, 

> When you're accessing `/assets/about/` without a file name specified, you'll receive a 403 since you're not allowed to see directory listings. At least this is the behavior I would expect and I get in my local docker setup (using the Dockerfile and docker-compose file from the repo).
> 
> However, I can't reproduce this with `fuesim-manv.de/assets/about/`, which shows me the Angular 404 page (although the HTTP status code is 403). Did you change some settings of nginx in deployment?

Yes, I changed it to this (removed uneccary commented out stuff):

```
    # Frontend
    root /usr/local/app/frontend/dist/digital-fuesim-manv;

     error_page 404 /index.html;
     error_page 403 /index.html;

    location /assets/about {
        # Requests to this location should not fall back to index.html
    }

    location / {
        try_files $uri $uri/ /index.html;
    }
...
```

Reply:
I like this approach since it it does not rely on a specific file (`non_existing_page.html`) to not exist. I've changed the config in https://github.com/hpi-sam/digital-fuesim-manv/pull/657/commits/3a51279e23ec41f501395dbcd277d1415c15b58a accordingly.